client/consensus/pow/src/lib.rs

```diff
@@ -170,6 +171,16 @@ pub trait PowAlgorithm<B: BlockT> {
 	) -> Result<Option<bool>, Error<B>> {
 		Ok(None)
 	}
+	/// Break a fork choice tie. By default this chooses the earliest block seen. Using uniform tie
+	/// breaking algorithms will help to protect against selfish mining. Returns if the new seal
+	/// should be considered best block.
+	fn break_tie(
+		&self,
+		_own_seal: &Seal,
```

Comment:
is it really `own_seal` or `old_seal`?

Reply:
`own_seal` means the seal we currently have. And `new_seal` is the seal we're about to import. I guess this is more referring to selfish mining (in which case one would always be choosing one's own seal).